router/broker_test.go

```diff
@@ -1311,4 +1311,23 @@ func TestEventHistory(t *testing.T) {
 	if ev1.Arguments[0].(int) < ev2.Arguments[0].(int) {
 		t.Fatalf("MetaRPC subEventHistory for topic %s should return records in reverse order", topic)
 	}
+
```

Comment:
You forgot to add the `reqId++` here. So the invocation below will be treated as a new request.